src/main/kotlin/com/levi/featurespitter/service/EndPointService.kt

```diff
@@ -0,0 +1,82 @@
+package com.levi.featurespitter.service
+
+import org.springframework.stereotype.Service
+import java.net.http.HttpClient
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
+import java.net.URI
+import com.levi.featurespitter.model.Store
+import com.levi.featurespitter.model.StoreSeason
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import org.springframework.beans.factory.annotation.Autowired
+
+@Service
+class EndPointService {
+
+
+	companion object {
+		fun getStores(): List<Store> {
+			val client = HttpClient.newBuilder().build();
+			val request = HttpRequest.newBuilder()
+				.uri(URI.create("http://134.209.29.209/v1/stores/?page=1"))
+				.header("apiKey", "nxzAAC1EQAAU7AGY8gUQnev_99A")
+				.header("accept", "application/json")
+				.build();
+
+
+			val response = client.send(request, HttpResponse.BodyHandlers.ofString());
+			println(response.body())
```

Comment:
If you wanted to persist these prints for future debugging, how would you do it?

Reply:
We can create api methods in controller to which some re-direction will happen in case of success or failure of other apis, which will trigger the service to save that log using its repository and entity.

Reply:
I was thinking of something even less sophisticated than that, like using log4j or slf4j to perform proper logging.

Reply:
Actually from your comment "persist these prints for future debugging", I thought we need to store logs in database for future debugging. Yes I understand, We can use log4j or slf4j logging utilities for proper logging.